packages/wrangler/src/sentry/index.ts

```diff
@@ -118,6 +119,19 @@ export function setupSentry() {
 							continue;
 						}
 						const nodeModulesIndex = frame.filename.indexOf("node_modules");
+
+						if (typeof jest !== undefined && nodeModulesIndex === -1) {
+							const wranglerPackageIndex = frame.filename.indexOf(
+								path.join("packages", "wrangler", "src")
+							);
+							if (wranglerPackageIndex === -1) {
+								continue;
+							}
+							frame.filename =
+								fakeInstallPath +
+								frame.filename.substring(wranglerPackageIndex);
+							continue;
+						}
```

Comment:
What's this for?

Reply:
This is because we snapshot against the stacktrace in some tests, and with the change to MSW interceptors the stack trace included the location of the Wrangler repo on disk. This code removes the Wrangler package's location from stack traces, only for testing